scripts/data_preprocess.py

```diff
@@ -0,0 +1,72 @@
+import click
```

Comment:
A description here as well:
```suggestion
Build preprocessor for data and saved transformed data.
import click
```